modules/data-entry/src/main/frontend/src/questionnaire/Form.jsx

```diff
@@ -211,18 +212,20 @@ function Form (props) {
     );
   }
 
-  let parentDetails = data?.subject?.parents && getHierarchy(data.subject.parents, React.Fragment, () => ({}));
+  let parentDetails = data?.subject?.parents && getHierarchy(data.subject, Link, (node) => ({href: "/content.html" + node["@path"], target :"_blank"}));
```

Comment:
I think this might fix the issue reported by Sasha (didn't test it)

```suggestion
  let parentDetails = data?.subject && getHierarchy(data.subject, Link, (node) => ({href: "/content.html" + node["@path"], target :"_blank"}));
```